opds.py

```diff
@@ -73,13 +73,17 @@ def __init__(self, _db, title, url, libraries, annotator=None,
 
         self.add_link_to_catalog(self.catalog, rel="self",
                                  href=url, type=self.OPDS_TYPE)
+        web_client_uri_template = ConfigurationSetting.sitewide(
+            _db, Configuration.WEB_CLIENT_URL
+        ).value
```

Comment:
Grabbing this out of the database before iterating over the list of libraries saves us one database query per library for the same data.